webapp/src/components/Analysis/UtterancesTable.tsx

```diff
@@ -376,7 +376,7 @@ const UtterancesTable: React.FC<Props> = ({
     <Box className={classes.gridContainer}>
       <div className={classes.gridHeaderActions}>
         <Description
-          text="View insights on individual utterances with details results and annotations."
+          text="Explore the utterances and propose actions. Click on a row to inspect the utterance details."
```

Comment:
I think I'd write either
```suggestion
          text="Explore the utterances in the selected subpopulation and propose actions. Click on a row to inspect the utterance details."
```

or

```suggestion
          text="Explore utterances and propose actions. Click on a row to inspect the utterance details."
```